--- debugedit-5.2/scripts/find-debuginfo.in.orig	2025-09-05 00:27:53.611458082 +0200
+++ debugedit-5.2/scripts/find-debuginfo.in	2025-09-05 00:31:29.180856030 +0200
@@ -462,9 +462,9 @@
 touch "$temp/primary"
 touch "$temp/linked"
 (
-    find "$RPM_BUILD_ROOT" ! -path "${debugdir}/*.debug" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) -print |
+    find "$RPM_BUILD_ROOT" ! -path "${debugdir}/*.debug" -type f -print |
         file -N -f - |
-        sed -n -e 's/^\(.*\):[     ]*.*ELF.*, not stripped.*/\1/p';
+        sed -n -e 's/^\(.*\):[     ]*.*ELF.*\(executable\|shared object\).*, not stripped.*/\1/p';
 
     # plus static libraries
     $process_ar && find "$RPM_BUILD_ROOT" -type f -name '*.a' -print |
