--- debugedit-5.0/scripts/find-debuginfo.in.orig	2021-07-27 01:05:31.867817624 +0200
+++ debugedit-5.0/scripts/find-debuginfo.in	2022-08-16 21:51:22.254424409 +0200
@@ -413,9 +413,8 @@
 # Build a list of unstripped ELF files and their hardlinks
 touch "$temp/primary"
 find "$RPM_BUILD_ROOT" ! -path "${debugdir}/*.debug" -type f \
-     		     \( -perm -0100 -or -perm -0010 -or -perm -0001 \) \
 		     -print | LC_ALL=C sort |
-file -N -f - | sed -n -e 's/^\(.*\):[ 	]*.*ELF.*, not stripped.*/\1/p' |
+file -N -f - | sed -n -e 's/^\(.*\):[ 	]*.*ELF.*\(executable\|shared object\).*, not stripped.*/\1/p' |
 xargs --no-run-if-empty stat -c '%h %D_%i %n' |
 while read nlinks inum f; do
   if [ $nlinks -gt 1 ]; then
